### DISCUSSION TITLE
How to properly separate argument list and pattern?

### DISCUSSION THREAD

**Discussion by stephane-archer** [BurntSushi/ripgrep#1989] [General] [answered]
with `rg -F "-cpus="` I have the following error message `error: Found argument '-=' which wasn't expected, or isn't valid in this context` but `-cpus=` is the pattern I look for.
How to tell rg that the last "argument" is actually the pattern I look for?

**Comment by BurntSushi** [comment, marked as answer]
From the man page:

```
       PATTERN
           A regular expression used for searching. To match a pattern beginning with a dash, use the
           -e/--regexp option.
```

So, `rg -F -e -cpus=`.

Alternatively, you can use the standard Unix convention of `--` to signal an end to non-positional arguments. So, `rg -F -- -cpus=`.
